.github/workflows/release.yml

```
@@ -0,0 +1,104 @@
+          echo "Tag v${{ steps.get_version.outputs.version }} already exists"
+        else
+          echo "exists=false" >> $GITHUB_OUTPUT
+          echo "Tag v${{ steps.get_version.outputs.version }} does not exist"
+        fi
+    
+    - name: Create extension zip
+      if: steps.check_tag.outputs.exists == 'false'
+      run: |
+        # Create a clean directory for the extension files
+        mkdir -p extension-package
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)